integreat_cms/static/src/js/media-management/index.tsx

```diff
@@ -89,6 +92,8 @@ const MediaManagement = (props: Props) => {
     const [refresh, setRefresh] = useState<boolean>(false);
     // This state contains a file which should be opened in the sidebar after the content has been refreshed
     const [sidebarFile, setSidebarFile] = useState<File>(null);
+    // whether the filter for unused media files is being activated
+    const [isUnsedFiltered, setFilterState] = useState<boolean>(false);
```

Comment:
I don't think we need a separate state for this - when we define a new route, the preact router automatically passes the query params to the rendered components.